File: Go/Basic/02-Data-Type/array.go
package main

import "fmt"

func main() {
	// array
	var names [3]string

	names[0] = "Petrus"
	names[1] = "Handika"
	names[2] = "Sinaga"

	fmt.Println(names[0])
	fmt.Println(names[1])
	fmt.Println(names[2])

	var value = [3]int {
		90, 80,
	}
	fmt.Println(value)

	// function array
	var values = [...]int {
		90,
		80,
		95,
	}
	fmt.Println(values)
	fmt.Println(len(values))
	values[0] = 100
	fmt.Println(values)
}

File: Java/Basic/Array.java
package Basic;

public class Array {
    public static void main(String[] args) {
        // Cara menginstansi array:
        // Deklarasi array
        // Inisialisasi array -> menggunakan operator new kemudian tambahkan tipe data yang sesuai dengan tipe data pada pendeklarasian
        // Menentukan ukuran array

        int [] arrayNum;

        int arrayNum2 [];

        int [] arrayNum3 = new int [5];

        arrayNum3[0] = 10;
        arrayNum3[1] = 20;
        arrayNum3[2] = 30;
        arrayNum3[3] = 40;
        arrayNum3[4] = 50;

        String [] ArrayString = {"Petrus", "Handika", "Nadia", "Mora"};
        System.out.println(ArrayString[0]);

        System.out.println(ArrayString.length);

        // Array multidimensi
        int [][] matrix = {
                {1, 2, 3},
                {4, 5, 6},
                {7, 8, 9},
        };
        System.out.println(matrix[2][2]);
    }
}
